atspi-common/src/events/object.rs

```diff
@@ -1004,7 +1059,18 @@ impl TryFrom<&zbus::Message> for ObjectEvents {
 			"BoundsChanged" => Ok(ObjectEvents::BoundsChanged(ev.try_into()?)),
 			"LinkSelected" => Ok(ObjectEvents::LinkSelected(ev.try_into()?)),
 			"StateChanged" => Ok(ObjectEvents::StateChanged(ev.try_into()?)),
-			"ChildrenChanged" => Ok(ObjectEvents::ChildrenChanged(ev.try_into()?)),
+			"ChildrenChanged" => {
+				let body: EventBodyOwned = ev.try_into()?;
+				match body.kind.as_str() {
```

Comment:
Are these discriminants correct?
In the old [LF event table](https://accessibility.linuxfoundation.org/a11yspecs/atspi/adoc/atspi-events.html) it says these minor strings were 'add' and 'remove' did that change?